[licensing] Updates for 5.1 (#7094)
sourcegraph/handbook: content/departments/technical-success/ce/process/license_keys.md

3. Follow the process below for generating a renewal license key. Unless otherwise noted during Step 1, use the same exact tags and user count as the existing key (make sure to comma separate each tag) and use the `"Active Contract Period End"` as the expiration date.
4. Send the new license key to the customer (follow the process outlined below).

## Revoking an active license

In extreme circumstances, it may be required to revoke a customer's active Sourcegraph license. This will mark the license as invalid on Sourcegraph.com, and license verification checks from the customer's Sourcegraph instance will fail, ultimately disabling all Sourcegraph Enterprise features.

To revoke an active license, follow these steps:

1. Sign in to sourcegraph.com and visit the **Site-admin > Subscriptions** page.
2. Navigate to the license that needs to be revoked.
3. Click on the red **Revoke** button and provide a reason for the revocation.

The next time the customer's instance performs a license verification check, the verification will fail and the customer's Enterprise features will be disabled.

> > Note: This will not work on licenses with the `allow-air-gapped` tag applied, as these license do not need to do any license verification checks.

Reviewer: I think we decided to use `enterprise-air-gap-0` plan instead of tag

Author: Gotcha, thanks!

Author: I'm going to mention both since I think it is technically possible to just apply that tag to any license

> > Note: This will not work on licenses with a license that has the `allow-air-gapped` tag applied or the `enterprise-air-gap-0` license, as these license do not need to do any license verification checks.

Reviewer 2: ```suggestion
> Note: This will not work on special licenses with the `allow-air-gapped` tag or the `enterprise-air-gap-0` plan, as these license are not executing verification checks.
```

- `plg-trial` to indicate that a Cloud trial managed instance has been requested through signup.sourcegraph.com.
- `dev` for internal developer licenses
- `internal` for licenses used for internal sites (dotcom, k8s, etc.)

> - `allow-air-gapped` for licenses where periodic license verification is not required. This is essential for customers that run Sourcegraph in a sandboxed environment without an external internet connection.

Reviewer 2: ```suggestion
- `allow-air-gapped` for licenses where periodic license verification is not required. This is essential for customers that run Sourcegraph in a sandboxed environment without an external internet connection. For `enterprise-air-gap-0` plan, the tag is not needed, since the plan already has the feature.
```
